src/components/form-draft/testing.vue

```diff
@@ -90,16 +90,20 @@ export default {
     // component is created.
     ...requestData([{ key: 'formDraft', getOption: true }, 'keys']),
     qrSettings() {
+      const url = apiPaths.serverUrlForFormDraft(
+        this.formDraft.draftToken,
+        this.projectId,
+        this.xmlFormId
+      );
       return {
-        server_url: apiPaths.serverUrlForFormDraft(
-          this.formDraft.draftToken,
-          this.projectId,
-          this.xmlFormId
-        )
+        general: { server_url: `${window.location.origin}${url}` },
+        project: {
+          name: this.$t('collectProjectName', this.formDraft),
+          icon: '📝'
+        },
+        // Collect requires the settings to have an `admin` property.
+        admin: {}
       };
-    },
-    baseUrl() {
```

Comment:
I randomly noticed that this computed property isn't used anywhere anymore! Probably it should have been removed as part of 43ceaa0c7ec9b74d3833c3398c9d8e2f7c4ca711.